firebase-config/src/main/java/com/google/firebase/remoteconfig/internal/ConfigRealtimeHttpClient.java

```diff
@@ -271,12 +283,17 @@ public void onEvent() {
           }
 
           @Override
-          public void onError(Exception error) {}
+          public void onError(Exception error) {
```

Comment:
Should we check this is the realtime disabled exception?

Reply:
As of right now this is only being onError method is only being used when there is a backoff response from the backend.